web/src/components/Inputs.js

```diff
@@ -1,5 +1,47 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Component } from 'react';
+import { Editor } from 'react-draft-wysiwyg';
+
+import { EDITOR_CONFIG, htmlToEditor, editorToHtml } from '../util/editor';
+
+class RichText extends Component {
+  constructor(props) {
+    super(props);
+
+    const { content, onSync } = props;
+
+    this.state = {
+      content: htmlToEditor(content)
+    };
+
+    this.onEditorChange = newContent => {
```

Comment:
what's the thinking behind these methods being inside the constructor?

Reply:
The thinking is "oops."  Hang tight, I'll put them where they belong.